lib/me.robot

```diff
@@ -0,0 +1,42 @@
+*** Comments ***
+@@ -0,0 +1,23 @@
+
+
+*** Settings ***
+Library     BuiltIn
+Library     Process
+
+
+*** Variables ***
+${EXPECTED}=    00:16.0
+
+
+*** Keywords ***
+Check ME Out
+    [Documentation]    Keyword Runs Lspci To Check Whether The ME Device
+    ...    Exists Or Not
+    ${out}=    List Devices In Linux    pci
```

Comment:
Just a thought/suggestion... This keyword is redundant. as it only glues together 'ls' with whatever we send to it. I think it is much more clear if we will use 'Execute Linux Command    lspci' instead. And in future maybe we should think about refactoring whole code to remove this keyword.

Reply:
true https://github.com/Dasharo/open-source-firmware-validation/pull/403/commits/cfba1a7ddeaae34ada8fb9ddf05d9319543e5516